Migration to multi-arch oc adm upgrade status reference (#85287)
openshift/openshift-docs: modules/migrating-to-multi-arch-cli.adoc

$ oc adm upgrade
----
+
.Example output

[source,terminal]
----

> working towards ${VERSION}: 106 of 841 done (12% complete), waiting on etcd, kube-apiserver

Reviewer: Recorded from our [test](https://redhat-internal.slack.com/archives/CJ1J9C3V4/p1732148310689569?thread_ts=1729872364.520849&cid=CJ1J9C3V4) :

```suggestion
working towards ${VERSION}: 761 of 890 done (85% complete), waiting on machine-config
```

It is likely that that "machine-config" got caught in the output because MCO is the only cluster operator that implement the method described in the [enhancement](https://github.com/openshift/enhancements/pull/1690).

----
+
[IMPORTANT]
====
Machine launches may fail as the cluster settles into the new state. To notice and recover when machines fail to launch, we recommend deploying machine health checks. For more information about machine health checks and how to deploy them, see _About machine health checks_.
====
+

> * If you want more detailed information about the status of your update, you can also monitor the migration by running the following Technology Preview command:

Reviewer 2: I feel like if you are introducing another step in this verification section, both steps now need to be numbered list items:
```suggestion
. If you want more detailed information about the status of your update, you can also monitor the migration by running the following Technology Preview command:
```

> * If you want more detailed information about the status of your update, you can also monitor the migration by running the following Technology Preview command:

Reviewer 2: Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps), steps always need to be written in the imperative mood (in this case that would mean removing "you can"). If this is an optional step, then [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps) suggests you start the step with "Optional:" and then still write the rest of the step in the imperative mood, as seen in the examples there.
```suggestion
* Optional: To retrieve more detailed information about the status of your update, monitor the migration by running the following Technology Preview command:
```

> * If you want more detailed information about the status of your update, you can also monitor the migration by running the following Technology Preview command:

Reviewer 2: Lastly, the tech preview aspect:

1. Up to you, but maybe adding a link to the page you mention below in the additional resources for this page would be helpful.
2. I am not sure, but is this an appropriate way to inform users that a command is technology preview (i.e. just mentioning technology preview as an adjective in the step)? I would personally expect to see the tech preview snippet below this step and not have TP mentioned in the step at all, but maybe there are examples of this kind of use case
```suggestion
* If you want more detailed information about the status of your update, you can also monitor the migration by running the following command:
```
